app/mailers/views/hypertension_fast_track_pilot_mailer.erb.html

```diff
@@ -6,11 +6,13 @@
   <body>
     <p>
       A single-issue <%= @disability_code %> claim for increase was submitted on va.gov.<br/>
-      <% if @rrd_claim_processed %>
+      <% case @description
+         when 'Processed' %>
         A health summary PDF was generated and added to the claim's documentation.
-      <% else %>
-        There was some issue (such as insufficient health data or pending EP) that prevents
-        a health summary PDF to be generated for this claim.
+      <% when 'Pending EP' %>
+        There was already a pending EP 020 for the veteran associated with this claim.
+      <% when 'Insufficient Data' %>
+        There was not sufficient data to generate a health summary PDF associated with this claim. 
```

Comment:
Perhaps a cleaner version of this:
```ruby
      <% case @rrd_status
         when :processed %>
        A health summary PDF was generated and added to the claim's documentation.
      <% when :pending_ep %>
        There was already a pending EP 020 for the veteran associated with this claim.
      <% when :insufficient_data %>
        There was not sufficient data to generate a health summary PDF associated with this claim. 
```
where `@rrd_status` is  returns a symbol like [Matt suggested](https://github.com/department-of-veterans-affairs/vets-api/pull/9480/files?diff=split&w=0#r836777828)